providers/amazon/tests/system/amazon/aws/example_sqs.py

```diff
@@ -61,12 +61,12 @@ def delete_queue(queue_url):
     publish_to_queue_1 = SqsPublishOperator(
         task_id="publish_to_queue_1",
         sqs_queue=sqs_queue,
-        message_content="{{ task_instance }}-{{ logical_date }}",
+        message_content="{{ task_instance }}",
```

Comment:
Were we previously trying to ensure that these messages were unique for some intentional reason? I don't think it matters, but just want to be sure.

Reply:
I wondered the same. To be honest I dont know but I ran the system test locally and it passed